src/fiskaltrust.ifPOS/v1/de/Models/FinishTransactionResponse.cs

```diff
@@ -6,34 +6,32 @@ namespace fiskaltrust.ifPOS.v1.de
     [DataContract]
     public class FinishTransactionResponse
     {
-        [DataMember(Order = 1)]
-        public long TransactionNumber { get; set; }
+        [DataMember(Order = 10)]
+        public ulong TransactionNumber { get; set; }
+
+        [DataMember(Order = 20)]
+        public DateTime StartTransactionTimeStamp { get; set; }
 
-        [DataMember(Order = 2)]
-        public DateTime StartTime { get; set; }
+        [DataMember(Order = 30)]
+        public DateTime TimeStamp { get; set; }
 
-        [DataMember(Order = 3)]
-        public DateTime EndTime { get; set; }
+        [DataMember(Order = 40)]
+        public string TseTimeStampFormat { get; set; }
 
-        [DataMember(Order = 4)]
-        public string CertificateSerialNumber { get; set; }
+        [DataMember(Order = 50)]
+        public string TseSerialNumberOctet { get; set; }
```

Comment:
Is this Base64, or really the stringified byte array ("00 03 1f...")?

Reply:
see also https://github.com/fiskaltrust/middleware-interface-dotnet/pull/4#discussion_r417943794

because of real octet string by definition i would stick to have it in the name, to be clear on usage.